client/src/reducers/text.js

```diff
@@ -0,0 +1,16 @@
+var TextReducer = (state = {
+	text: null
+}, action) => {
+	switch (action.type) {
+	case 'ADD_TEXT':
+		return {
+			...state,
+			text: action.text
+		};
+	default: {
+		return state;
+	}
```

Comment:
Unneeded braces. Usually I wouldn't care but having two `}` indented at the same level may look buggy on a first glance.

Reply:
Mightve been added by ESLint

Reply:
Probably not, since the other `case` did not have braces.

Reply:
@TimothyGu Agh, sorry, just saw this now; how would I go about updating after merging?

Reply:
@dustinnewman98 You can file a new pull request :wink: